### DISCUSSION TITLE
state/ period

### DISCUSSION THREAD

**Discussion by EriHau** [revk/xESP32-Faikout#181] [Q&A] [answered]
I'm a bit confused with the status messages and don't understand the manual: In the manual, the settings say 

> reporting | Interval for reporting state (seconds)

 However, the interval for "state/" is not set, instead the interval for "Faikin/" is set when I change this setting. The manual also says 

> state/ | topic indicate current state, and are reported **periodically** and on some state changes.

I can't recognize any period. Before I switched on the "livestatus", sometimes there was no new message for the state/ topic for more than half an hour. With the "livestatus" it is better, but if the device is not running, a message is also very rare. With what max period the "state/" topic should be sent?

**Comment by revk** [comment]
I can look through code and check.

This has evolved over years, so some things are sent in several different ways.

**Reply by EriHau** [reply]
That would be very nice. Because for all clients that do not support a last will statement, I have a timeout monitoring in the topics to react accordingly in my controller when the client dies.

**Reply by revk** [reply]
There is definitely a "will" status set.

**Reply by EriHau** [reply]
And for which topic and what would be the message?

**Reply by revk** [reply, marked as answer]
Well, my guess is state/name as a top level status message that shows when up. But let me check...

Yes, here, I see a will of:

state/GuestAC {"up":false}

Hope that helps.

**Reply by EriHau** [reply]
Perfect! Then I will use this topic for my monitoring! Thank you very much for your quick support. **You are doing such a great job here**!!!
